apps/frontend-v3/app/(marketing)/risks/page.tsx

```diff
@@ -982,9 +983,24 @@ export default function Privacy() {
                     <p>
                       Oracles are data providers which supply external information to smart
                       contracts. Oracles, like Chainlink, may be used to source exchange rates
-                      between pool tokens for a rate provider in Balancer MetaStable pools. The
-                      risks of using Oracles to supply exchange rates include:
+                      between pool tokens for a rate provider in Balancer MetaStable pools.
+                    </p>
+                    <p>
+                      Arbitrageurs can exploit oracle latency by frontrunning oracle updates at
+                      direct expense to liquidity providers. If an arbitrageur sees an oracle update
+                      in the mempool that will significantly change an asset’s price, they can
```

Comment:
`asset&apos;` (there are probably more of these)


&lsquo;timelock delays&rsquo;

Reply:
I think it is not using traditional quotes so it is no problem for the linter, etc.